### ISSUE TITLE
Widget has fillers

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5-image#19]
![image](https://cloud.githubusercontent.com/assets/156149/21266646/564f80ae-c3a7-11e6-9799-737bf169fcc6.png)

**Comment by szymonkups** [comment]
I've pushed my progress to [t/19 branch](https://github.com/ckeditor/ckeditor5-image/tree/t/19). We decided that we will leave this for now as the issue is bigger than it looked at the beginning.
Problem is that sometimes browser put selection inside `contenteditable="false"` element. I wanted to listen on `selectionChange` event and fix the selection before it is stored in model. Unfortunately, this causes some side-effects:
![output](https://cloud.githubusercontent.com/assets/2396463/22886911/3a11d20c-f200-11e6-8b4f-b978f4cf458f.gif)

This happens:
1. Selection is dragged, it ends up inside nested editable, so it's fixed to select whole widget. 
1. Selection is converted to model selection.
1. Model to view selection conversion is done, widget is selected, and selection is put inside fake selection container.
1. Selection is dragged further so 1st step occurs again.

In short, we are in a process of dragging selection in browser and at the same time we want to put selection inside fake selection container.

**Comment by Reinmar** [comment]
Another TC which we identified to be caused by the filler: https://github.com/ckeditor/ckeditor5-image/issues/69.

**Comment by Reinmar** [comment]
Another TC: https://github.com/ckeditor/ckeditor5/issues/489.

**Comment by pjasiun** [comment]
Fixed by https://github.com/ckeditor/ckeditor5-engine/pull/1173.
